cpp/tests/abm_helpers.cpp

```diff
@@ -49,14 +49,16 @@ void interact_testing(mio::abm::PersonalRandomNumberGenerator& personal_rng, mio
 {
     // allocate and initialize air exposures with 0
     mio::abm::AirExposureRates local_air_exposure;
-    local_air_exposure.resize({mio::abm::CellIndex(location.get_cells().size()), mio::abm::VirusVariant::Count});
```

Comment:
have you checked that there are no other relevant resizes that need to be changed?

Reply:
Yes, we don't resize CustomIndexArray that much anyways. Also, if it were used incorrectly somewhere, the compiler would complain, vehemently.